src/Canvas.cc

```diff
@@ -720,14 +718,7 @@ Canvas::~Canvas() {
 	}
 }
 
-std::vector<FontFace>
-_init_font_face_list() {
-  std::vector<FontFace> x;
-  return x;
-}
-
-std::vector<FontFace> Canvas::_font_face_list = _init_font_face_list();
-
+std::vector<FontFace> Canvas::_font_face_list;
```

Comment:
I'm a little worried I made `_init_font_face_list` for some esoteric reason but it doesn't look necessary. That said do we even need the green line?

Reply:
Yeah I was wondering if at some point `_init_font_face_list` was being called more than once, but couldn't find evidence of that.

The green line is necessary (static members have to be initialized in the source and not the header so they're not redefined).

Reply:
Although I don't really see a reason why it needs to be a member of `Canvas`; it could probably just be in the source file unless you know of a reason it might be needed externally?

Reply:
It could be in the source file yeah, as long as there's one list per process since that's how system fonts get registered. Will merge as-is or with that change if you want it.

Reply:
Move it to the source file (since I messed up the newline after it originally 😛). Should be ready to go!

If you and @LinusU are preparing to release, #1337 fixes an important regression and should be merged as well.